modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/ExportedMissingReferenceBackwardCompatbilityExportImportTest.java

```diff
@@ -143,12 +143,12 @@ public void testBackwardCompatibility() throws Exception {
 
 			@Override
 			public void evaluate() throws Throwable {
-				Stream<Method> methodStream = _parentTestMethods.stream();
+				List<Method> methods = ListUtil.filter(
+					_parentTestMethods,
+					m -> Objects.equals(
+						m.getName(), description.getMethodName()));
 
-				Assume.assumeTrue(
-					methodStream.noneMatch(
```

Comment:
@JanisZhang In https://github.com/liferay-core-infra/liferay-portal/pull/3662, you converted some usages to anyMatch; one of them should be suitable for `noneMatch`. Please also apply to that.